deployment/deployment_test.go

```diff
@@ -529,6 +534,78 @@ func TestAirflowUpgradeCancelError(t *testing.T) {
 	assert.Error(t, err, "API error (500):")
 }
 
+func TestAirflowUpgradeEmptyDesiredVersion(t *testing.T) {
+	testUtil.InitTestConfig()
+	okResponse := `{"data": {
```

Comment:
can you please fix formating of json using 2 spaces and remove tabs?

Reply:
Done.